scripts/DB.js

```diff
@@ -1,48 +1,109 @@
 /**
  * Connections to the Database are made here
  */
-import patients from "../stubs/patients";
-import tasks from "../stubs/tasks";
-// Note: Look into pools? https://www.npmjs.com/package/mysql#pooling-connections
+require("dotenv").config();
+var mysql = require("mysql");
+const DBConnection = require("./DBConnection");
 
-const DB = {
-  /** ----- Tasks ----- */
-  retrieveTask: function (taskID) {
-    const task = tasks[taskID];
-    if (patient === undefined)
-      throw Error(`Task '${taskID}' does not exist in the Database`);
+var connection = mysql.createConnection({
```

Comment:
Use the `DBConnection ` which contains the DB host, user, password, and database instead of connection